src/routes/asset.php

```diff
@@ -82,7 +81,16 @@
             $order_column = $column_mapping[$params['sort']];
         }
     }
+
+    // Sorting should be reversed by default only for dates, not strings
+    $reverse = isset($params['sort']) ? $params['sort'] === 'updated' : false;
```

Comment:
As there already is an `if isset($params['sort'])` check right above, `$reverse` could be initialized to `false` before and then `$reverse = $params['sort'] === 'updated';` be placed inside this if block.